core/admin.py

```
@@ -0,0 +1,3 @@
+from aplicaciones.usuario.admin import *
```

Ruff (0.8.2)

`from aplicaciones.usuario.admin import *` used; unable to detect undefined names

(F403)